depends/hosts/darwin.mk

```diff
@@ -0,0 +1,17 @@
+OSX_MIN_VERSION=10.7
+OSX_SDK_VERSION=10.9
+OSX_SDK=$(SDK_PATH)/MacOSX$(OSX_SDK_VERSION).sdk
+LD64_VERSION=241.9
```

Comment:
This is the actual version we use. Bitcoin for OSX is cross-built from Linux in Gitian. LD64 comes from native_cctools, see here: https://github.com/theuni/cctools-port